package.json

```diff
@@ -36,17 +36,18 @@
     "url": "git://github.com/opentable/spur-common.git"
   },
   "dependencies": {
-    "spur-ioc": "^0.2.1",
-    "spur-errors": "^0.2.1",
-    "spur-config": "^1.0.0",
-    "lodash": "^3.0.0",
     "bluebird": "^2.9.3",
-    "winston": "^0.8.3",
     "superagent": "^3.0.0",
-    "moment-timezone": "^0.3.0",
+    "colors": "1.1.0",
     "form-data": "^0.2.0",
     "joi": "^5.1.0",
-    "colors": "1.1.0"
+    "lodash": "^4.0.0",
```

Comment:
Can you please use `^4.17.4`? I want the history to reflect when we switched and to what version.